addons/sale/tests/test_sale_prices.py

```diff
@@ -777,7 +777,7 @@ def test_fiscalposition_application(self):
             line.product_id = product_tmpl_c.product_variant_id
             line.product_uom_qty = 1.0
         sale_order = order_form.save()
-        self.assertRecordValues(sale_order.order_line, [{'price_unit': 100, 'price_subtotal': 84.91}])
+        self.assertRecordValues(sale_order.order_line, [{'price_unit': 100, 'price_subtotal': 84.34}])
```

Comment:
why? how did that affect the use case?

Reply:
input: [10 fixed incl, 15%] on 100.0
FP: 15% => 6% incl

Before: 100 with [10 incl, 6% incl] = (100 - 10) / 1.06 = 84.91
After: 100 / 1.06 - 10 = 84.34

Since both taxes are included in price, the last one should be evaluated first so now, the % is removed from the price first.